### PR TITLE
Add memcached module with stats metricset

### PR DESCRIPTION
**Pull request by ruflin**
A new module "memcached" was added which gathers metrics from https://memcached.org/. The `stats` command is used in the `stats` metricset to gather all infos about the service.

### CODE REVIEW

**Comment by ruflin** [comment]
jenkins, retest it

**Comment by ruflin** [comment]
jenkins, retest it

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -98,6 +98,7 @@ https://github.com/elastic/beats/compare/v5.1.1...master[Check the HEAD diff]
 - The Docker, Kafka, and Prometheus modules are now Beta, instead of experimental. {pull}3525[3525]
 - The HAProxy module is now GA, instead of experimental. {pull}3525[3525]
 - Add the ability to collect the environment variables from system processes. {pull}3337[3337]
+- Add memcached module with stats metricset.
```
Comment (tsg) [inline CHANGELOG.asciidoc]: Link to PR please :)

File: metricbeat/docs/fields.asciidoc [hunk 2]
```diff
@@ -3321,6 +3322,139 @@ type: keyword
...
+[float]
+=== memcached.stats.pid
+
+type: long
+
+Current process ID of the Memcached task.
+
+
+[float]
+=== memcached.stats.uptime
```
Comment (tsg) [inline metricbeat/docs/fields.asciidoc]: Is this seconds? Should we have it as `uptime.secs` then?

File: metricbeat/module/memcached/stats/stats.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	}
+}
+
+type MetricSet struct {
+	mb.BaseMetricSet
+}
+
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+
+	logp.Warn("BETA: The docker container metricset is beta")
```
Comment (tsg) [inline metricbeat/module/memcached/stats/stats.go]: s/docker/memcached

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ruflin) [inline CHANGELOG.asciidoc]: added

File: metricbeat/docs/fields.asciidoc (hunk 2 above)
Comment (ruflin) [inline metricbeat/docs/fields.asciidoc]: yes, changed it to `uptime.sec`.

**Pull request merged**